src/app/home/home.component.ts

```diff
@@ -12,7 +12,10 @@ import { UserInputDialogComponent } from '../user-input-dialog/user-input-dial.c
     styleUrls: ['./home.component.css'],
 })
 export class HomeComponent {
-    constructor(private dialog: MatDialog, private route: Router) {}
+    constructor(
+        private dialog: MatDialog,
+        private route: Router,
+    ) {}
```

Comment:
did you use the npm format?
when i use it i will get this code `constructor(private dialog: MatDialog, private route: Router) {}` in one line
and more changes even in index.html like that `<!doctype html>`